StreamChat.xcodeproj/project.pbxproj

```diff
@@ -8824,12 +10312,12 @@
 				minimumVersion = 1.8.2;
 			};
 		};
-		DB823105259C271C00E7D7FE /* XCRemoteSwiftPackageReference "SwiftyGif" */ = {
+		C1BE72712732C91F006EB51E /* XCRemoteSwiftPackageReference "Nuke" */ = {
```

Comment:
Will this change keep the `Nuke` reference in the project?

Reply:
Nuke is used in the DemoApp, as well as other example projects. The idea behind this is to find any possible clashes

Reply:
Ah. ok. I see your point ;)